packages/rspack-plugin-react-refresh/src/index.ts

```diff
@@ -71,3 +71,6 @@ export default class ReactRefreshRspackPlugin {
 }
 
 ReactRefreshRspackPlugin.deprecated_runtimePaths = runtimePaths;
+
+// @ts-expect-error output module.exports
+export = ReactRefreshRspackPlugin;
```

Comment:
this seems a breaking change

Reply:
Why is this a breaking change, it will output:

```js
ReactRefreshRspackPlugin.deprecated_runtimePaths = runtimePaths;
module.exports = ReactRefreshRspackPlugin;
```

Same as the original js file.

Reply:
oh, I thought original is export default

Reply:
![image](https://github.com/web-infra-dev/rspack/assets/8898718/8f3111eb-f822-4f2c-b6f5-fc2d5a01a77a)
this is misleading

Reply:
haha, GitHub's fault